src/website/views/components/time-picker/_examples.pug

```diff
@@ -75,17 +62,12 @@ p.-text
         </div>
       </div>
 
-      <script>chi.timePicker(document.getElementById('unique-id-di1'));</script>
-
-h3 Value, min and max options
+h3 Value option
 .example.-mb--3
   .-p--3
     div.chi-form__item(style="max-width: 14rem;")
-      label.chi-label(for='unique-id-va1')
-        | Time
-      div.chi-input__wrapper.-icon--right
-        input.chi-input(type="time" id="unique-id-va1")
-        i.chi-icon.icon-circle-clock-outline.-icon--muted
+      chi-label(for='unique-id-va1') Time
+      chi-time-picker#unique-id-va1(value='09:00:00')
```

Comment:
Minor inconsistency - value has seconds but seconds column is not displayed in the drop down menu